src/clojure_tictactoe/tictactoe.clj

```diff
@@ -1,31 +1,78 @@
 (ns clojure-tictactoe.tictactoe
-  (:gen-class))
+  (:gen-class)
+  (:require [clojure.string :as string]))
 
 
-(def welcome-message 
+(def welcome-message
   "Welcome to the game of TicTacToe! \n")
 
-(def instructions-message 
+(def instructions-message
  "Instructions: 2 players take turns placing markers, try to get three of your markers in a row to win! \n
 
-Examples: 
+Examples:
 
   Vertical      Horizontal      Diagonal
 
-   | x |     |    |   |     |  x |   |     
-===+===+===  | ===+===+===  | ===+===+===  
-   | x |     |  x | x | x   |    | x |     
-===+===+===  | ===+===+===  | ===+===+===  
+   | x |     |    |   |     |  x |   |
+===+===+===  | ===+===+===  | ===+===+===
+   | x |     |  x | x | x   |    | x |
+===+===+===  | ===+===+===  | ===+===+===
    | x |     |    |   |     |    |   | x  \n\n ")
 
 
 (defn print-game-intro []
   (println welcome-message)
   (println instructions-message))
 
+;TODO make this DRYer when you have minute and a better understanding of Clojure
 (defn board [spaces]
```

Comment:
Would it make more sense to call this `render-board` rather than just board? Because it's not actually storing any board related information.

Reply:
I like that!